### PR TITLE
Styled landing page

### PR DESCRIPTION
**Pull request by deetolman**

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Home.css';
+
```
Comment (acrankin) [inline src/components/Home.js:4]:
```suggestion
```

File: src/components/Home.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  background: #272822;
+  color: #F8F8F2;
+  letter-spacing: 0.3px;
+  word-spacing: 1px;
+  :link:hover {
+    background: gray;
+    color: white;
+    text-decoration: none;
+  }
+
```
Comment (acrankin) [inline src/components/Home.css]:
```suggestion
```

**Comment by acrankin** [comment]
Looks good, just delete those extra lines

File: src/components/Home.js (hunk 1 above)
Comment (deetolman) [reply src/components/Home.js:4]: Done!

**Comment by acrankin** [comment]
go forth and merge!

**Pull request merged**